v2/emailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -46,10 +48,117 @@ If you navigate to `/auth`, you should not see the widget anymore.
 
 ## Step 2: Render the component yourself
 
```

Comment:
need to use the right component as used on this page: https://deploy-preview-756--admiring-bhabha-7b1be9.netlify.app/docs/thirdpartyemailpassword/common-customizations/signup-form/adding-fields

Reply:
done